runtime/doc/news.txt

```diff
@@ -335,6 +335,9 @@ The following changes to existing APIs or features add new behavior.
 
 • The `workspace/didChangeWatchedFiles` LSP client capability is now enabled
   by default.
+  • On Mac or Windows, `libuv.fs_watch` is used as the backend.
+  • On Linux, `fswatch` is used a the backend if available, otherwise
```

Comment:
Should we maybe also mention here that `fswatch` is recommended?
```suggestion
  • On Linux, `fswatch` (recommended) is used a the backend if available, otherwise
```